lib/src/accordion/types.ts

```diff
@@ -0,0 +1,69 @@
+type Space = "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
+type Margin = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
+type Padding = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
+type SVG = HTMLElement & SVGElement;
+
+type Props = {
+  /**
+   * The panel label.
+   */
+  label?: string;
+  /**
+   * Represents the state of the panel. When true, the component will be
+   * expanded. If undefined, the component will be uncontrolled and its
+   * value will be managed internally by the component.
+   */
+  isExpanded?: boolean;
+  /**
+   * Element used as the icon that will be placed next to panel label.
+   */
+  icon?: SVG;
+  /**
+   * @deprecated URL of the icon that will be placed next to panel label.
+   */
+  iconSrc?: string;
+  /**
+   * Assistive text to be placed on the right side of the panel.
+   */
+  assistiveText?: string;
+  /**
+   * If true, the component will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * This function will be called when the user clicks the accordion to expand or collapse
+   * the panel. The new state of the panel will be passed as a parameter.
+   */
+  onChange?: (isExpanded: boolean) => void;
+  /**
+   * The expanded panel of the accordion. This area can be used to render
+   * custom content.
+   */
+  children: React.ReactNode;
```

Comment:
Same. I think this prop should be optional.

Reply:
Prop shouldn't be optional in order to enforce the design guidelines.